classes/music_album.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(publish_date, on_spotify: true)
+    super(publish_date)
```

Comment:
nice implementation of the music_album class

Reply:
Thank you @RichardChileya 👏